config/octodns.yaml

```diff
@@ -0,0 +1,19 @@
+---
+providers:
+  config:
+    class: octodns.provider.yaml.YamlProvider
+    directory: ./config
+    default_ttl: 3600
+    enforce_order: True
+  route53:
+    class: octodns_route53.Route53Provider
+    access_key_id: env/AWS_ACCESS_KEY_ID
+    secret_access_key: env/AWS_SECRET_ACCESS_KEY
+
+zones:
+  dsd.io.:
+    lenient: true
```

Comment:
lenient flag = "I know what I'm doing and accept any problems I run across."

Is this wise?

Reply:
This has to be true as octodns demands a trailing ".", which r53 cannot cater for.

Reply:
Should I add a comment?